src/test/memory-leak/runMemlabTests.js

```
@@ -30,4 +31,9 @@ const consoleMode = 'VERBOSE';
 
     runResult.cleanup();
   }
-})();
+}
+
+
+runMemlab().catch(err => {
+  console.error('Ошибка при запуске memory lab tests:', err);
```

ESLint

[error] Unexpected console statement.

(no-console)